Sources/AppStoreConnectClient/AppStoreConnectError.swift

```diff
@@ -29,13 +29,31 @@ public enum AppStoreConnectError: Error, Equatable {
     /// A server-side error occurred, indicated by the provided status code.
     case serverError(errorCode: Int)
     /// No results were found for the specified bundle ID.
-    case unauthorized
+    case unauthorized(errors: String?)
     /// The request is forbidden.
-    case forbidden
+    case forbidden(errors: String?)
     /// The request is malformed or contains invalid parameters.
-    case badRequest
+    case badRequest(errors: String?)
     /// The requested resource was not found.
-    case notFound
-    /// The credentials is empty.
-    case noHaveCredentials
+    case notFound(errors: String?)
+    /// An unexpected error occurred.
+    case unexpectedError(errors: String?)
+}
+
+extension AppStoreConnectError {
```

Comment:
Move to a separate file AppStoreConnectError+Extensions

Reply:
Moved to separate file.